packages/fast-css-editor-react/webpack.config.js

```diff
@@ -0,0 +1,56 @@
+const path = require("path");
+const ForkTsCheckerWebpackPlugin = require("fork-ts-checker-webpack-plugin");
+const HtmlWebpackPlugin = require("html-webpack-plugin");
+
+const appDir = path.resolve(__dirname, "./app");
+const outDir = path.resolve(__dirname, "./www");
+
+module.exports = {
+    devtool: "inline-source-map",
+    entry: path.resolve(appDir, "index.tsx"),
+    output: {
+        path: outDir,
+        publicPath: "/",
+        filename: "[name].js"
+    },
+    mode: process.env.NODE_ENV || "development",
+    module: {
+        rules: [
+            {
+                test: /.tsx?$/,
+                use: [
+                    {
+                        loader: "ts-loader",
+                        options: {
+                            transpileOnly: true
+                        }
+                    }
+                ]
+            }
+        ]
+    },
+    plugins: [
+        new ForkTsCheckerWebpackPlugin({
+            tslint: path.resolve(__dirname, "../../tslint.json")
+        }),
+        new HtmlWebpackPlugin({
+            contentBase: outDir,
+        })
+    ],
+    resolve: {
+        extensions: [".js", ".tsx", ".ts", ".json"],
+        alias: {
+            fbjs: path.resolve('./node_modules/fbjs'),
+            'lodash-es': path.resolve('./node_modules/lodash-es'),
+            react: path.resolve('./node_modules/react'),
+            'react-dom': path.resolve('./node_modules/react-dom'),
+        }
+    },
```

Comment:
Does this get run through webpack for production independently of an app? If we don't need to package an "App" instance of this, can we remove all the alias resoltion from 40-48?

Reply:
Yes, we need this for our build to check on the app.